src/source/Response.c

```diff
@@ -282,78 +282,6 @@ VOID terminateCurlSession(PCurlResponse pCurlResponse, UINT64 timeout)
     }
 }
 
-VOID dumpResponseCurlEasyInfo(PCurlResponse pCurlResponse)

```

Comment:
This is a diagnostic / debug function I think we should keep it. Is there a reason why we're deleting it (other than it's not currently used and is decreasing our overall code line coverage count -- I don't think those are good reasons to delete it).  Instead if that is the reason can we just add a call to this function hidden behind a flag of some sort? If the function itself is buggy then that's a different situation.

Reply:
As discused offline, added this back and commented it out.